core/templates/services/context.service.ts

```diff
@@ -251,10 +256,23 @@ export class ContextService {
       ServicesConstants.PAGE_CONTEXT.COLLECTION_EDITOR,
       ServicesConstants.PAGE_CONTEXT.TOPIC_EDITOR,
       ServicesConstants.PAGE_CONTEXT.STORY_EDITOR,
-      ServicesConstants.PAGE_CONTEXT.SKILL_EDITOR
+      ServicesConstants.PAGE_CONTEXT.SKILL_EDITOR,
+      ServicesConstants.PAGE_CONTEXT.TOPICS_AND_SKILLS_DASHBOARD
```

Comment:
Why do we need this?

Reply:
Since otherwise the complex RTE elements (like images, collapsibles, tabs and so on) weren't shown in the editor. There is a check in one of the ck-editor files that calls this function.

Reply:
I see, thanks for the info.